docs/advanced/secure-setup.md

```diff
@@ -0,0 +1,315 @@
+* All details/commands that come within curly braces `{}` cannot be copy-pasted,
+  you will need to remove the curly braces and edit them to suit your setup.
+
+### Benefits of this Setup
+
+* The wallet `Wallet-B` is kept offline and turned off most of the time.
+  This allows greatly reduces attack surface/opportunity.
+* An attacker would first have to compromise the dcrd on `Computer A` and find a
+  way to jump into `Wallet-B` using the RPC to both compromise and exfiltrate
+  data.
+  This is highly unlikely and difficult to pull off.
+* dcrd on `Computer A` can be used for other purposes in your local network (eg. other wallets, DEX, VSPD, etc),
+  so you don't need to maintain multiple dcrd instances within your local network.
+
+### Setting up Computer A
+
+The best way to setup decred on your computer is to use [dcrinstall](../wallets/cli/cli-installation.md)
+
+**Note: If you setup dcrwallet in this system do not use the same seed and passwords for your Wallet-B**
+
+Once this is done it is assumed that you have dcrd bins and the dcrd config files.
+
+Note down the IP address of `Computer-A` on your local network.
+
+`ifconfig`
+
+eg:192.148.1.105
+
+You should start dcrd `Computer-A` and let it sync to tip.
+You should also ensure it exposes RPC to the local network.
+
+`dcrd --rpclisten=localip`
+
+eg: `dcrd --rpclisten=192.148.1.105`
+
+We should now start setting up a folder that will be copied over to Wallet-B
+
+`mkdir ~/copytob`
+
+Copy the certificate and config.
+
+```
+cp ~/.dcrd/rpc.cert ~/copytob
+cp ~/.dcrd/dcrd.conf ~/copytob
+```
+
+**Windows users can follow along by using the relevant folders from [here](../wallets/cli/os-differences.md#application-directory-locations) and using 7z for tar.**
+
+---
+
+=== "Raspberry Pi"
+
+    [Download](https://github.com/decred/decred-binaries/releases/download/v{{ cliversion }}/decred-linux-arm64-v{{ cliversion }}.tar.gz) the decred bins for arm64 (Raspberry Pi) and place them in the same folder :
+
+
+    `wget -P ~/copytob https://github.com/decred/decred-binaries/releases/download/v{{ cliversion }}/decred-linux-arm64-v{{ cliversion }}.tar.gz`
+
+=== "General systems"
+
+    [Download](https://github.com/decred/decred-binaries/releases/download/v{{ cliversion }}/decred-linux-amd64-v{{ cliversion }}.tar.gz) the decred bins for linux and place them in the same folder :
+
+    `wget -P ~/copytob https://github.com/decred/decred-binaries/releases/download/v{{ cliversion }}/decred-linux-amd64-v{{ cliversion }}.tar.gz`
+
+---
+
+Make sure you [verify](verifying-binaries.md) the tar file.
+
+Now you can tar the  `copytob` folder for transport.
+
+`tar -zcvf ~/copytob.tar.gz -C ~/copytob decred-linux-{arm/amd}64-v{{ cliversion }}.tar.gz dcrd.conf rpc.cert`
+
+You can also calculate the hash to ensure that the file is not modified while moving.
+
+`sha256sum ~/copytob.tar.gz`
+
+Store this output on an uneditable medium. ie: paper or a photograph.
```

Comment:
Why store this hash on paper/photo? Seems like a strange suggestion to me, I think we should justify it.

Reply:
If we ask people to just store it people might save the hash into the computer itself. Which ofc can be manipulated by any malicious program.